test/integration_test/common_test.go

```diff
@@ -44,15 +44,15 @@ const (
 	// defaultValidateDeployTimeout is a default timeout for deployment validation
 	defaultValidateDeployTimeout = 900 * time.Second
 	// defaultValidateDeployRetryInterval is a default retry interval for deployment validation
-	defaultValidateDeployRetryInterval = 30 * time.Second
+	defaultValidateDeployRetryInterval = 5 * time.Second
```

Comment:
Was this for internal testing or you generally wanted to make the polling faster?